integration-tests/src/test/java/uk/gov/di/authentication/api/AuthenticationCallbackHandlerIntegrationTest.java

```diff
@@ -627,7 +624,7 @@ private void validateClaimsInJar(SignedJWT signedJWT) throws java.text.ParseExce
         assertThat(
                 signedJWT.getJWTClaimsSet().getClaim("govuk_signin_journey_id"),
                 equalTo(CLIENT_SESSION_ID));
-        assertThat(signedJWT.getJWTClaimsSet().getClaim("vtr"), equalTo(List.of("P2", "P1")));
```

Comment:
P1 is currently not supported - when rebased there will be other supported LoCs which can be used in this test to parse lists